website/layouts/index.html

```diff
@@ -0,0 +1,69 @@
+{{ partial "header" . }}
+
+<body class="home">
+	<section id="hero">
+		<header>
+			<div class="wrapper">
+				<a href="/" id="logo"><img src="img/logo.svg"></a>
+			</div>
+		</header>
+		<div class="wrapper">
+			<div class="hero-content">
```

Comment:
This content is not part of e.g. Safari's reader view, which it is on https://docs.gateway.clearhaus.com/ right now. We probably want this "Welcome" part to be included in the reader view in this new version as well.

I haven't tested, but this might also affect screen readers.